pydoctor/test/test_astbuilder.py

```diff
@@ -71,17 +71,37 @@ def unwrap(parsed_docstring: ParsedEpytextDocstring) -> str:
     assert isinstance(value, str)
     return value
 
+class NotFoundLinker(DocstringLinker):
+    """A DocstringLinker implementation that cannot find any links."""
+
+    def resolve_identifier(self, identifier: str) -> Optional[str]:
+        return None
+
+    def resolve_identifier_xref(self, identifier: str, lineno: int) -> str:
+        raise LookupError(identifier)
+
 def to_html(parsed_docstring: ParsedDocstring) -> str:
-    return flatten(parsed_docstring.to_stan(None))
+    return flatten(parsed_docstring.to_stan(NotFoundLinker()))
 
-def type2str(type_expr: object) -> Optional[str]:
+@overload
+def type2str(type_expr: None) -> None: ...
+
+@overload
+def type2str(type_expr: ast.expr) -> str: ...
+
+def type2str(type_expr: Optional[ast.expr]) -> Optional[str]:
     if type_expr is None:
         return None
     else:
         src = astor.to_source(type_expr)
         assert isinstance(src, str)
         return src.strip()
 
+def str_and_line(obj: model.Documentable) -> Tuple[str, int]:
```

Comment:
maybe rename `annotation_type_and_line` ... and add a short docstring. :)

Reply:
I'd like to keep the name short to keep the asserts that use it readable. But I can rename it to `ann_str_and_line` to be a bit more descriptive.

I'll add a docstring.

Reply:
or just `ann_str_line` 

I am old school, and I prefer to write my own assertion helper... like `def assertStrAndLine(self, name, line, actual):`
and have all the ugly processing inside the method.

Reply:
A custom assert method would fit well in the unittest/trial model, but with the way pytest reports test failures, I prefer to have the asserts outside the helper functions (when feasible).